src/feasibility_checker.jl

```diff
@@ -0,0 +1,221 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+using LinearAlgebra
+
+function _last_primal_solution(model::Model)
+    if !has_values(model)
+        error(
+            "No primal solution is available. You must provide a point at " *
+            "which to check feasibility."
+        )
+    end
+    return Dict(v => value(v) for v in all_variables(model))
+end
+
+"""
+    primal_feasibility_report(
+        model::Model,
+        point::AbstractDict{VariableRef,Float64} = _last_primal_solution(model),
+        atol::Float64 = 0.0,
+        skip_missing::Bool = false,
+    )::Dict{Any,Float64}
+
+Given a dictionary `point`, which maps variables to primal values, return a
+dictionary mapping the constraint reference of each constraint in `model` to the
+distance between the point and the nearest feasible point, if the distance is
+greater than `atol`.
+
+## Notes
+
+ * If `skip_missing = true`, constraints containing variables that are not in
```

Comment:
This docstring does not say what happens of `skip_missing` is `false` and the value of some variables is not given

Reply:
Added